components/board/pi/impl/digital_interrupts.go

```diff
@@ -197,13 +197,6 @@ func (i *ServoDigitalInterrupt) Tick(ctx context.Context, high bool, now uint64)
 	return nil
 }
 
-// AddCallback currently panics.
-func (i *ServoDigitalInterrupt) AddCallback(c chan board.Tick) {
```

Comment:
If you have a `ServoDigitalInterrupt` and want to add a callback, what are you supposed to do with it now? You can't call `AddCallback` from line 118 because the type of `i` is wrong, and I don't believe there's any way to cast a `ServoDigitalInterrupt` into a `BasicDigitalInterrupt` because they are independent types.

Reply:
I don't think you ever could, this originally panics, no?

Reply:
:facepalm: Very good point, my mistake!